code/game/machinery/civilian_bounties.dm

```diff
@@ -87,7 +87,14 @@
 		if(curr_bounty.applies_to(AM))
 			active_stack ++
 			curr_bounty.ship(AM)
-			qdel(AM)
+			if(ismachinery(AM))
+				var/obj/machinery/machine_bounty = AM
+				for(var/obj/part in machine_bounty.component_parts)
+					qdel(part)
+				qdel(machine_bounty.circuit)
+				qdel(machine_bounty)
```

Comment:
TBH, `qdel(machine_bounty)` should be the only thing needed here.

Reply:
I really hope it's the only thing needed here

Reply:
Well, I did try that, but I tested it to verify, and just the qdel leaves the parts and circuit, minus the frame.
![image](https://user-images.githubusercontent.com/41715314/99437743-dc600e00-28e0-11eb-9a69-316314939989.png)

Reply:
This is not a perfect world apparently